lib/utils.pm

```diff
@@ -600,7 +601,7 @@ sub zypper_call {
                     $0; if ($0 ~ /statistics/ ){ print "EOL"; group++ }; }\'\
                     /var/log/zypper.log
                     ';
-    for (1 .. 5) {
+    for (1 .. ($retry or 1)) {
```

Comment:
Will this ever evaluate to 1 .. 1? Doesn't $retry default to 5?

Reply:
see https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/20383#discussion_r1797238569